cmd/feed-ingress/main.go

```diff
@@ -62,6 +71,19 @@ func configureLogging() {
 	}
 }
 
+func configureHealthPort() {
+	http.HandleFunc("/health", checkHealth)
```

Comment:
I'll check.

Also, do you think we should wait for nginx to start up before returning healthy?

Should we distinguish between ready and liveness?